src/NServiceBus.Transport.SQS.AcceptanceTests/When_trying_to_subscribe_when_topic_does_not_exist.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NServiceBus.AcceptanceTesting.Customization;
+    using NServiceBus.AcceptanceTesting.Support;
+    using NServiceBus.Features;
+    using NUnit.Framework;
+
+    public class When_trying_to_subscribe_when_topic_does_not_exist : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_log_exception_with_topic_name()
+        {
+            var messageId = Guid.NewGuid();
+
+            await Scenario.Define<Context>(c =>
+            {
+                c.MessageId = messageId;
+                c.EnableInstallers = true;
+                c.DisableAutoSubscribe = true;
+            })
+            .WithEndpoint<Endpoint>()
+            .Done(context => true)
+            .Run();
+
+            var context = await Scenario.Define<Context>(c =>
+            {
+                c.MessageId = messageId;
+                c.EnableInstallers = false;
+                c.DisableAutoSubscribe = false;
+            })
+            .WithEndpoint<Endpoint>()
+            .Done(context => true)
+            .Run();
+
+            Assert.IsTrue(context.Logs.Any(a => a.Message.Contains("not found") && a.Message.Contains("When_trying_to_subscribe_when_topic_does_not_exist-MyEvent")));
+        }
+
+        public class Context : ScenarioContext
+        {
+            public Guid MessageId { get; set; }
+            public bool EnableInstallers { get; set; }
+            public bool DisableAutoSubscribe { get; set; }
+        }
+
+        public class Endpoint : EndpointConfigurationBuilder
+        {
+            public Endpoint() =>
+                EndpointSetup<DefaultServerWithNoInstallers>((e, c) =>
+                {
+                    var ourContext = c.ScenarioContext as Context;
```

Comment:
testContext or scenarioContext instead of ourContext?

Reply:
renamed